PowerPointLabs/Test/FunctionalTest/ShapesLabTest.cs

```diff
@@ -15,14 +17,15 @@ public class ShapesLabTest : BaseFunctionalTest
         private const int AddShapesShapesSlide = 6;
         private const int AddShapesTestSlide = 7;
         private const int AddShapesExpSlide = 8;
+        private const int AddShapesPlaceholder = 9;
```

Comment:
Hi @YuPeiHenry, can you rename this to `AddShapesPlaceholderSlide` instead for consistency?

Reply:
@ChesterSng Thanks for the correction! I have changed the naming to be consistent.